client/lib/service-worker/service-worker.js

```diff
@@ -106,21 +106,21 @@ self.addEventListener( 'message', function( event ) {
 
 /* eslint-disable */
 self.addEventListener( 'fetch', function( event ) {
-	if ( event.request.mode === 'navigate' ) {
-		event.respondWith(
-			caches.open( CACHE_VERSION ).then( function( cache ) {
-				return cache.match( event.request ).then( function( response ) {
-					return (
-						response ||
-						fetch( event.request ).then( function( response ) {
+	event.respondWith(
```

Comment:
@Tug Once we decide exactly which assets to cache, it may be worth including logic here that shortcuts the cache lookup, which can actually be kind of expensive (depending on the platform). e.g. limiting to GETs, or certain mime types, etc. You don't need to change anything right now - just an FYI since you removed the navigation check.

Reply:
I added this check back and some others :)